### PR TITLE
[WIP] MMTL bleu score

### PR DESCRIPTION
**Pull request by bhancock8**
Putting @jdunnmon's branch for adding bleu-based scores in a PR so we can see diffs.

### CODE REVIEW

File: metal/mmtl/task.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def __init__(
         head_module,
         scorer=Scorer(standard_metrics=[]),
         loss_hat_func=(
-            lambda Y_prob, Y_gold: F.mse_loss(torch.sigmoid(Y_prob), Y_gold)
+            lambda Y_prob, Y_gold: F.mse_loss(torch.sigmoid(Y_prob), Y_gold.float())
```
Comment (jdunnmon) [inline metal/mmtl/task.py]: @bhancock8 should this be fixed somewhere else?  I was getting a weird long-float error and just casted it away...

File: metal/mmtl/dataset.py [hunk 2]
```diff
@@ -84,14 +84,14 @@ def __init__(
                 True: (tokens, segment), labels
                 False: tokens, labels
         """
-        tokenizer = BertTokenizer.from_pretrained(bert_model, do_lower_case=True)
+        self.tokenizer = BertTokenizer.from_pretrained(bert_model, do_lower_case=True)
```
Comment (bhancock8) [inline metal/mmtl/dataset.py]: why do we need to assign to self here?

File: metal/mmtl/launch.py [hunk 3]
```diff
@@ -71,6 +80,12 @@ def get_dir_name(models_dir):
             "This is used, e.g., for tuning."
         ),
     )
+
+    # Use auxiliarty tasks
```
Comment (bhancock8) [inline metal/mmtl/launch.py]: auxiliarty :)

File: metal/mmtl/launch.py [hunk 4]
```diff
@@ -71,6 +80,12 @@ def get_dir_name(models_dir):
...
+    parser.add_argument(
+        "--use_auxiliary", type=int, default=0, help="Use auxiliary tasks or not"
```
Comment (bhancock8) [inline metal/mmtl/launch.py]: I'd make the default Bool, just to be clear. It'll still accept 0 and 1 as values.

File: metal/mmtl/task.py (hunk 1 above)
Comment (bhancock8) [reply metal/mmtl/task.py]: Yeah...suspicious this hasn't come up elsewhere before...can you mark this with a TODO or HACK note so we know to pay attention to it?

File: metal/mmtl/dataset.py (hunk 2 above)
Comment (jdunnmon) [reply metal/mmtl/dataset.py]: Because we need it to be able to take the dataset and re-compute new labels for it when the new task (e.g. BLEU score) requires the original tokens -- we need the inds_to_tokens method to get the raw strings back, b/c lots of useful NLP utilities (e.g. NLTK) need the raw tokens.

File: metal/mmtl/launch.py (hunk 3 above)
Comment (jdunnmon) [reply metal/mmtl/launch.py]: Derp. Fixed.

File: metal/mmtl/launch.py (hunk 4 above)
Comment (jdunnmon) [reply metal/mmtl/launch.py]: Done
